backend/tracim_backend/tests/functional/test_user.py

```diff
@@ -4968,7 +4992,48 @@ def test_api__set_user_info__ok_200__admin(self):
         assert res['user_id'] == user_id
         assert res['profile'] == 'administrators'
 
-    def test_api__set_user_info__err_403__user_itself(self):
+    def test_api__set_user_profile__err_403__admin_itself(self):
```

Comment:
`s/403/400/`

Reply:
@inkhey you check a code=400 in the code and the name of the test is 403 => ce n'est pas cohérent.

Par ailleurs le cas est le suivant : 

- si un utilisateur admin essaie de changer son profil, c'est un 400 => c'est un cas d'utilisation qui n'a pas de sens
- un utilisateur autre essaie de changer son profil, c'est une 403 => il n'est pas autorisé à changer le profil d'un utilisateur.

error.USER_CANT_CHANGE_IS_OWN_PROFILE